notebooks/239-freevc/239-freevc.ipynb

```diff
@@ -0,0 +1,761 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.    ir_net_g_model = mo.convert_model(onnx_path, compress_to_fp16=True)
</pre><p>now, conversion step happends each time when notebook run, it can be quite time consuming if we just want use notebook for running inference, could you please make conversion is optional (if model already converted and stored on drive)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>